src/site/topic21.rst

```diff
@@ -34,6 +34,50 @@ Tree Definition
    :align: center
 
 
+
+Relationships
+-------------
+
+* A **parent/predecessor** of a given node is the node directly above in the hierarchy
+    * Each node can have at most one parent, except for the root, which has no parent
+
+* A **child/successor** of a given node is the node directly below in the hierarchy
+    * Each node can have between zero and many children
```

Comment:
_between zero and many_ sounds odd. _at least zero_ doesn't work... maybe simply saying nodes can have any number of children? The context kind of implies non-negative and integers.

Reply:
agreed.